### PR TITLE
Moves css rules from links.ts to links.css

### PR DESCRIPTION
**Pull request by dmartzol**
For: https://github.com/microsoft/vscode/issues/165169

- [x] `src/vs/editor/contrib/links/browser/links.ts`.

### CODE REVIEW

**Comment by dmartzol** [comment]
@microsoft-github-policy-service agree [company="default"]

**Comment by dmartzol** [comment]
@microsoft-github-policy-service agree

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/vs/editor/contrib/links/browser/links.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
 .monaco-editor .detected-link-active {
 	text-decoration: underline;
 	text-underline-position: under;
+	color: var(--vscode-activeLink-foreground) !important;
```
[the remark this answers is not in the document]
Comment (dmartzol) [reply src/vs/editor/contrib/links/browser/links.css]: ahh, thank you! Fixed it

**Review by aeschli — DISMISSED** [review]

**Comment by aeschli** [comment]
Thanks @weartist !

**Review by aeschli — APPROVED** [review]

**Review by dbaeumer — APPROVED** [review]

**Pull request merged**